lib/pacemaker/pcmk_sched_clone.c

```diff
@@ -522,29 +521,26 @@ probe_anonymous_clone(pe_resource_t *clone, pe_node_t *node)
  * \return true if any probe was created, otherwise false
  */
 bool
-clone_create_probe(pe_resource_t *rsc, pe_node_t *node)
+pcmk__clone_create_probe(pe_resource_t *rsc, pe_node_t *node)
 {
-    CRM_ASSERT(rsc);
+    CRM_CHECK((node != NULL) && pe_rsc_is_clone(rsc), return false);
 
     rsc->children = g_list_sort(rsc->children, pcmk__cmp_instance_number);
-    if (rsc->children == NULL) {
```

Comment:
impossible based on unpack, right?

Reply:
Correct, clone_unpack() always creates at least one instance to correspond to any orphans found